net/MapViewController.m

```diff
@@ -395,23 +359,14 @@ -(BOOL)tabBarController:(UITabBarController *)tabBarController shouldSelectViewC
     if ([viewController isKindOfClass:[UINavigationController class]] && [viewController.restorationIdentifier isEqualToString:@"description"]){
```

Comment:
I haven't already implemented notification to update map from description. So for now I still using Map delegate :(